### PR TITLE
feat: conditional coloring for weight table, weight reset button, add RMSE & R squared, documentation update

### PR DESCRIPTION
**Pull request by malwina0**

### CODE REVIEW

File: pages/home.py [hunk 1]
```diff
@@ -333,6 +336,15 @@ def update_model(contents, filename, df, column, about_us):
     return children, children, weights_plots, model_names, predictions, task, proba_predictions, weights
 
 
+#
```
Comment (jakub-piw) [inline pages/home.py:339]: Is this comment important?

**Comment by jakub-piw** [comment]
The weights annotation is missing.

**Review by jakub-piw — APPROVED** [review]
The weights component looks and works perfectly. The documentation looks much better.

File: pages/home.py (hunk 1 above)
Comment (malwina0) [reply pages/home.py:339]: It's a question to @D3TR0

Comment (D3TR0) [reply pages/home.py]: It was unnecessary

**Review by D3TR0 — APPROVED** [review]
I have no objections to the changes

**Pull request merged**